public/Aframe/samples/loading-scene-1/components.js

```diff
@@ -0,0 +1,127 @@
+AFRAME.registerComponent("collision-detect", {
```

Comment:
I think this component can be removed completely right? It only seems to be adding an event listener that does a console.log